src/components/TvDetails/index.tsx

```diff
@@ -615,6 +617,12 @@ const TvDetails: React.FC<TvDetailsProps> = ({ tv }) => {
                 )}
               </div>
             )}
+            {data.originalName && data.originalLanguage !== locale.slice(0, 2) && (
```

Comment:
are the local languages _always_ two-character codes?

Reply:
```suggestion
            {data.originalName && data.originalLanguage.startsWith(locale.slice(0,2)) && (
```
maybe?

Reply:
API documentation here: https://developers.themoviedb.org/3/configuration/get-languages